x/auth/client/cli/tx_sign.go

```diff
@@ -59,7 +59,10 @@ account key. It implies --signature-only.
 
 	flags.AddTxFlagsToCmd(cmd)
 
-	cmd.MarkFlagRequired(flags.FlagFrom)
+	err := cmd.MarkFlagRequired(flags.FlagFrom)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Let's be consistent, and in general we shouldn't just print the error like that.
Below you've discarded the error, so we can do it here too. Or handle the ones below.